data_steward/validation/metrics/required_labs.py

```diff
@@ -0,0 +1,179 @@
+                                                  CHECK_REQUIRED_LAB_QUERY)
+
+LOGGER = logging.getLogger(__name__)
+
+MEASUREMENT_CONCEPT_SETS_TABLE = 'measurement_concept_sets'
+MEASUREMENT_CONCEPT_SETS_DESCENDANTS_TABLE = 'measurement_concept_sets_descendants'
+
+MEASUREMENT_CONCEPT_SETS_FIELDS = [{
+    "type": "string",
+    "name": "panel_name",
+    "mode": "nullable",
+    "description": ""
+}, {
+    "type": "integer",
+    "name": "panel_omop_id",
+    "mode": "nullable",
+    "description": ""
+}, {
+    "type": "string",
+    "name": "measurement_name",
+    "mode": "required",
+    "description": ""
+}, {
+    "type": "integer",
+    "name": "measurement_omop_id",
+    "mode": "required",
+    "description": ""
+}]
+
+MEASUREMENT_CONCEPT_SETS_DESCENDANTS_FIELDS = [{
+    "type": "integer",
+    "name": "panel_omop_id",
+    "mode": "nullable"
+}, {
+    "type": "string",
+    "name": "panel_name",
+    "mode": "nullable"
+}, {
+    "type": "integer",
+    "name": "measurement_concept_id",
+    "mode": "nullable"
+}, {
+    "type": "string",
+    "name": "measurement_concept_name",
+    "mode": "nullable"
+}, {
+    "type": "integer",
+    "name": "ancestor_concept_id",
+    "mode": "nullable"
+}, {
+    "type": "string",
+    "name": "ancestor_concept_name",
+    "mode": "nullable"
+}, {
+    "type": "string",
+    "name": "classification",
+    "mode": "nullable"
+}, {
+    "type": "integer",
+    "name": "descendant_concept_id",
+    "mode": "nullable"
+}, {
+    "type": "string",
+    "name": "descendant_concept_name",
+    "mode": "nullable"
+}, {
+    "type": "string",
+    "name": "descendant_concept_class_id",
+    "mode": "nullable"
+}]
+
+
+def load_required_lab_table(project_id, dataset_id):
+    """
+    Loads the required lab table from resources/measurement_concept_sets.csv
+    into project_id.ehr_ops
+
+    :param project_id: Project where the dataset resides
+    :param dataset_id: Dataset where the required lab table needs to be created
+    :return: None
+    """
+
+    try:
+        LOGGER.info(
+            'Upload {measurement_concept_sets_table}.csv to {dataset_id} in {project_id}'
+            .format(
+                measurement_concept_sets_table=MEASUREMENT_CONCEPT_SETS_TABLE,
+                dataset_id=dataset_id,
+                project_id=project_id))
+
+        bq_utils.load_table_from_csv(project_id,
+                                     dataset_id,
+                                     MEASUREMENT_CONCEPT_SETS_TABLE,
+                                     csv_path=None,
+                                     fields=MEASUREMENT_CONCEPT_SETS_FIELDS)
+
+    except (oauth2client.client.HttpAccessTokenRefreshError,
+            googleapiclient.errors.HttpError):
+
+        LOGGER.exception("FAILED:  CSV file could not be uploaded:\n%s",
+                         app_identity)
+
+
+def load_measurement_concept_sets_descendants_table(project_id, dataset_id):
```

Comment:
Are these tables loaded during `generate_metrics` in `validation/main.py`?

Reply:
Those tables are not loaded in `generate_metrics`, and they have been loaded as an one-off. I added those functions just in case we need to load them again. But I should check if those tables exist in `ehr_ops`, if not create those.